src/platform/site-wide/maintenance-banner/components/App.js

```diff
@@ -0,0 +1,84 @@
+// Node modules.
+import React, { Component } from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import classnames from 'classnames';
+import moment from 'moment';
+// Relative imports.
+import environment from 'platform/utilities/environment';
+import localStorage from 'platform/utilities/storage/localStorage';
+import config from '../config';
+
+const {
+  content,
+  expiresAt,
+  id,
+  startsAt,
+  title,
+  warnContent,
+  warnStartsAt,
+  warnTitle,
+} = config;
```

Comment:
It would be more idiomatic to pass these in as props. 

We should fetch this information from pager duty using something like `getCurrentGlobalDowntime` in `src/platform/monitoring/DowntimeNotification/util/helpers.js`. The parent component for this could transform that data into the view props (e.g. I think you only get the downtime window but you'd need to turn that into stuff like `warnStartsAt`). 

If we use that helper then we don't need to do a site deploy to turn this on- we can set a maintenance window for the global downtime in PagerDuty.

Reply:
Oh nice! I'll check that out 🙂